diktat-gradle-plugin/build.gradle.kts

```diff
@@ -3,7 +3,7 @@ import org.gradle.nativeplatform.platform.internal.DefaultNativePlatform.getCurr
 
 plugins {
     `java-gradle-plugin`
-    kotlin("jvm") version "1.7.0"
+    kotlin("jvm") version "1.7.10"
```

Comment:
need to check, probable it can be removed after migration to gradle 7.5.*

Reply:
@petertrr, we used `"1.4.21"` here to be compatible with old version of gradle. do we need to upgrade this version?

Reply:
We are setting `apiVersion` and `languageVersion` for this build, so as long as `apiVersion = 1.3` is supported by Kotlin plugin, we can upgrade